draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -152,49 +164,69 @@ Streaming video contains the occasional key frame ("I-frame") containing a full
 These are necessary to rebuild receiver state after loss of delta frames.
 The key frames are therefore more critical to deliver to the receiver than delta frames.
 
-Streaming video also contains audio frames which can be encoded separately and thus can be signaled separately.
-Audio is more critical than video for almost all applications, but its importance (relative to other packets in the flow) is still an application decision.
+Streaming video also contains audio frames which can be encoded
+separately and thus can be signaled separately (requirement:
+REQ-MEDIA-KEYFRAME).
+
+Audio is more critical than video for almost all applications, but its
+importance (relative to other packets in the flow) is still an
+application decision (requirements: REQ-MEDIA-AV-SEPARATE,
+REQ-MEDIA-CLIENT-DECIDES).
+
+Especially with media over QUIC, the server or proxy sends the same
+stream to all receivers, including the same metadata.  Thus, when a
+client needs different prioritization (e.g., video over audio), this
+is only achievable by signaling that priority inversion from the
+client to the ISP router (requirement: REQ-CLIENT-DECIDES).
```

Comment:
Does this assume the host is always directly connected to the ISP network?

Reply:
No, it assumes the client can tell the ISP router the priority of the incoming flow as the client wants.  The client could be behind a NAT.

Reply:
I would zoom into that part to avoid confusion.  We can reason about "upstream router (home router, ISP router, etc.)" to cover both direct and indirect cases